public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingGroup.tsx

```diff
@@ -110,66 +157,35 @@ function ConditionalRenderingGroupRenderer({ model }: SceneComponentProps<Condit
         />
       </Field>
 
-      <Divider spacing={1} />
-
-      {value.map((entry) => (
-        <Fragment key={entry!.state.key}>
-          {/* @ts-expect-error */}
-          <entry.Component model={entry} />
+      <Stack direction="column" gap={2}>
+        {value.map((entry) => entry.render())}
 
-          <div className={styles.entryDivider}>
-            <Divider spacing={1} />
-            <p className={styles.entryDividerText}> {condition}</p>
-            <Divider spacing={1} />
-          </div>
-        </Fragment>
-      ))}
+        {viewNewRule ? (
+          <Select
+            allowCustomValue={false}
+            placeholder={t('dashboard.conditional-rendering.group.add.placeholder', 'Select rule type')}
+            options={newRuleOptions}
+            onChange={({ value }) => model.addItem(value!)}
+          />
+        ) : null}
+      </Stack>
 
       <div className={styles.addButtonContainer}>
-        <Dropdown
-          overlay={
-            <Menu>
-              <Menu.Item
-                label={t('dashboard.conditional-rendering.group.add.data', 'Data')}
-                onClick={() => model.addItem(new ConditionalRenderingData({ value: true }))}
-              />
-              <Menu.Item
-                label={t('dashboard.conditional-rendering.group.add.interval', 'Interval')}
-                onClick={() => model.addItem(new ConditionalRenderingInterval({ value: '7d' }))}
-              />
-              <Menu.Item
-                label={t('dashboard.conditional-rendering.group.add.variable', 'Variable value')}
-                onClick={() =>
-                  model.addItem(new ConditionalRenderingVariable({ value: { name: '', operator: '=', value: '' } }))
-                }
-              />
-            </Menu>
-          }
-        >
-          <ToolbarButton icon="plus" iconSize="xs" variant="canvas">
-            <Trans i18nKey="dashboard.conditional-rendering.group.add.button">Add condition based on</Trans>
-          </ToolbarButton>
-        </Dropdown>
+        <Button icon="plus" variant="secondary" size="sm" fullWidth onClick={() => setViewNewRule(true)}>
+          <Trans i18nKey="dashboard.conditional-rendering.group.add.button">Add rule</Trans>
+        </Button>
```

Comment:
Can you use the ValuePicker here instead? it transforms into a Select that opens when you click it, that way you do not have to hit a but that moves down and adds a select you then have to click. 

https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/components/PanelEditor/getFieldOverrideElements.tsx#L214